enterprise/utils.py

```diff
@@ -1000,29 +997,17 @@ def _get_service_worker(service_worker_username):
         return None
 
 
-def discovery_query_url(content_filter, html_format=True):
+def get_enterprise_catalog_preview_url(uuid, html_format=True):
     """
-    Return discovery url for preview.
+    Get the url for the preview information for an enterprise catalog
     """
-    if CatalogIntegration is None:
-        raise NotConnectedToOpenEdX(
-            _(
-                'To get a CatalogIntegration object, this package must be '
-                'installed in an Open edX environment.'
-            )
-        )
-    discovery_root_url = CatalogIntegration.current().get_internal_api_url()
-    disc_url = '{discovery_root_url}{search_all_endpoint}?{query_string}'.format(
-        discovery_root_url=discovery_root_url,
-        search_all_endpoint='search/all/',
-        query_string=urlencode(content_filter, doseq=True)
-    )
+    preview_url = EnterpriseCatalogApiClient.API_BASE_URL + EnterpriseCatalogApiClient.GET_CONTENT_METADATA_ENDPOINT.format(uuid)
```

Comment:
then here, i would recommend creating a static function in `EnterpriseCatalogApiClient` that returns the preview url so you can mock it in this method and test that function separately. it brings up the question of whether this function is necessary; why not call the function on the client directly?  that's an option too; but at the end of the day the caller will need to make sure `EnterpriseCatalogApiClient` is mocked out in the test, and the function returning the url is tested separately.

Reply:
Sounds good, I'll do that.